step2-06/README.md

```diff
@@ -1,27 +1,29 @@
-# Step 2.6
+# Step 2.6 - Redux: Dispatching actions and examining state
 
 [Lessons](../) | [Exercise](./exercise/) | [Demo](./demo/)
 
-Redux: Dispatching Actions and Examining State.
+In this step, we learn about the Redux methods `dispatch()` and `getState()`. Dispatching action messages to the store is the only means by which to instruct the reducers to modify the shared state tree.
 
-In this step, we learn about `dispatch` and `getState()`. Dispatching action messages to the store is the only means by which to inform the reducers to modify the shared state tree.
+We also see how to compose reducers to make up the complete state shape.
 
-We also see how we may compose the reducers according to the shape.
+## Dispatch
 
-# Dispatch
-
-As long as there is a store reference (we'll look at how to pass the store and the dispatch function into the view later), you can dispatch an action to trigger the middleware and reducers which in term changes the store which in turn will cause re-renders in the view.
+Given a store reference, you can dispatch an action to trigger the middleware and reducers. This changes the store and causes the view to re-render. (We'll look at how to pass the store and the dispatch function into the view later.)
 
 ```ts
-const store = createStore(redcuers);
+const store = createStore(reducers);
 store.dispatch(actions.addTodo('id0', 'hello world'));
 ```
 
-> Note: It is important to note that dispatches in general have a "fire and forget" approach. We expect React to re-render the UI correct on its own accord. Rendering isn't necessarily synchronous in React! Chaining async action creators is a topic for Step 9
+> Important note: Dispatches generally have a "fire and forget" approach. We expect React to re-render the UI correctly of its own accord. (Rendering isn't necessarily synchronous in React! Chaining async action creators is a topic for Step 9.)
+
+## Reducers scoped to a portion of the state tree
+
+In general, when an application grows, so does the complexity of the state tree. In a Redux application, it is best to have reducers that deal with only a sub-portion of the tree.
 
-# Reducers scoped to a portion of the state tree
+In our example, we have two parts of our state: `todos` and `filter`. We will split the reducer to pass the todos to a `todosReducer()` function and just return `all` to the `filter` key for now. This organization helps in navigating and understanding the reducers because it matches the shape of the state one-to-one: there's a separate reducer for each key in state.
```

Comment:
I wonder if "composition" would be a better term than "organization"? But it might be more confusing to people with less programming background.

Reply:
yeah let's keep it at org for now?